mimic/imimic.py

```diff
@@ -46,7 +70,51 @@ def catalog_entries(tenant_id):  # pragma:nocover
 
     def uri_for_service(region, service_id):  # pragma:nocover
         """
-        Get the URI for the service in a given region
+        Get the URI for the service in a given region.
+        """
+
+    def list_tenant_endpoints(tenant_id):  # pragma:nocover
+        """
+        List the tenant specific endpoints.
+        """
+
+    def enable_endpoint_for_tenant(tenant_id, template_id):  # pragma:nocover
+        """
+        Enable an endpoint for a specific tenant.
+        """
+
+    def disable_endpoint_for_tenant(tenant_id, template_id):  # pragma:nocover
+        """
+        Disable an endpoint for a specific tenant.
+        """
+
+    def list_templates():  # pragma:nocover
+        """
+        List the available templates.
+        """
+
+    def add_template(endpoint_template):  # pragma:nocover
+        """
+        Add a new template for the external API.
+
+        :param unicode templates: a :obj:`IEndpointTemplate` to add to the
+            :obj:`IExternalAPIMock` instance
+        """
+
+    def update_template(endpoint_template):  # pragma:nocover
+        """
+        Update an existing template for the external API.
+
+        :param unicode templates: a :obj:`IEndpointTemplate` to add to the
+            :obj:`IExternalAPIMock` instance
+        """
+
+    def remove_template(template_id):  # pragma:nocover
+        """
+        Add a new template for the external API.
```

Comment:
This docstring should probably `remove an existing template from the external API` or something similar.